vocab/vocab.csv

```diff
@@ -696,6 +721,13 @@ CP05MOAS-GL374-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 374,3-W
 CP05MOAS-GL374-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 374,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
 CP05MOAS-GL374-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 374,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
 CP05MOAS-GL374-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 374,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
+CP05MOAS-GL375,Coastal Pioneer,Mobile Assets,Coastal Glider 375,,,,TBD,TBD
+CP05MOAS-GL375-00-ENG000000,Coastal Pioneer,Mobile Assets,Coastal Glider 375,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,TBD,TBD
+CP05MOAS-GL375-01-ADCPAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 375,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,TBD,TBD
+CP05MOAS-GL375-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 375,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,TBD,TBD
+CP05MOAS-GL375-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 375,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),TBD,TBD
+CP05MOAS-GL375-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 375,Dissolved Oxygen,Aanderaa,Optode 4831,TBD,TBD
```

Comment:
What is TBD, TBD? Should this be 0, 1000?

Reply:
Ah good catch Peter - since we're touching this csv for the the other updates, we might as well update the GL375 rows to 0, 1000.